chat-plugins/uno.js

```diff
@@ -152,12 +152,13 @@ class UNOgame extends Rooms.RoomGame {
 		this.room.add(msg).update();
 	}
 
-	getPlayers(showCards) {
+	getPlayers(showCards, hashColor) {
```

Comment:
If were going to automatically show the color on usernames we don't really need to pass an extra variable do we? We can just automatically do it. And i posted this review before you pmed me on PS.